### PR TITLE
Delete your account

### PR DESCRIPTION
**Pull request by tamarasuarezrod**
## Description
How many animations do you want? Yes 🙂

**This PR includes:**
- Delete your account screen where you have to tap the correct circle in order to delete it. I also added some floating dogs just for fun and more distraction 🐶.
- Types for the Button component.
- Upgrade Reanimated to the latest version because I wanted to use the `withClamp` built-in animation, which was added in the last version.

https://github.com/rootstrap/react-native-bad-ux-challenge/assets/14322814/8b39608a-e518-4b05-8406-ddc411768953

image

### CODE REVIEW

File: src/components/DeleteAccountButton/styles.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import {StyleSheet} from 'react-native';
+
+const SEIZE = 100;
```
Comment (jpirazusta) [inline src/components/DeleteAccountButton/styles.ts]:
```suggestion
const SIZE = 100;
```

**Review by jpirazusta — APPROVED** [review]
🚀 🚀

File: src/components/DeleteAccountButton/styles.ts (hunk 1 above)
Comment (ManuViola77) [inline src/components/DeleteAccountButton/styles.ts]:
```suggestion
const SIZE = 100;
```

I think you have a typo in this name

**Review by ManuViola77 — APPROVED** [review]
Awesome work 🎉  Love the idea 💜  🚀

**Pull request merged**